### PR TITLE
Adding NEW TAKE button to the content list.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -210,6 +210,25 @@ define([
...
+                .activity('New Take', {
+                    label: gettext('New Take'),
+                    icon: 'copy',
+                    filters: [{action: 'list', type: 'archive'}],
+                    privileges: {archive: 1},
+                    condition: function(item) {
+                        return (item.lock_user === null || angular.isUndefined(item.lock_user)) && item.state !== 'killed';
+                    },
+                    controller: ['data', '$rootScope', 'desks', 'authoring', 'notify', function(data, $rootScope, desks, authoring, notify) {
+                        authoring.linkItem(data.item, null, null || desks.getCurrentDeskId())
```
Comment (plamut) [inline client/app/scripts/superdesk-archive/module.js]: The second `null` is redundant, the expression always evaluates to `desks.getCurrentDeskId()`.

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk-archive/module.js (hunk 1 above)
Comment (mdhaman) [inline client/app/scripts/superdesk-archive/module.js]: correct and fixed.

Comment (plamut) [inline client/app/scripts/superdesk-archive/module.js]: Now that was fast.

:+1:

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**